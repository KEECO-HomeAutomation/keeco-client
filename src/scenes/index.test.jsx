import React from 'react';
import { shallow } from 'enzyme';

import Scenes from './index';

describe('<Scenes />', () => {
	test('Should render without error', () => {
		const comp = shallow(<Scenes />);
		expect(comp.length).toBe(1);
	});

	test('Should match snapshot', () => {
		expect(shallow(<Scenes />)).toMatchSnapshot();
	});
});
