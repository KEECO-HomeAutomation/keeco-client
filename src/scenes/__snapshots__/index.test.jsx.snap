// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scenes /> Should match snapshot 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/setServer"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/login"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(withRouter(branch(branch(withHandlers(WithStyles(Dashboard))))))",
          "type": [Function],
        }
      }
      path="/"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
